tasks/set_boot_order_director.yml

```diff
@@ -0,0 +1,77 @@
+---
+- block:
```

Comment:
NIT: instead of writing the whole playbook inside the block, we can add the when condition in the caller task